vault/capabilities.go

```diff
@@ -37,6 +37,45 @@ func (c *Core) Capabilities(token, path string) ([]string, error) {
 		policies = append(policies, policy)
 	}
 
+	if te.EntityID != "" {
+		entity, err := c.identityStore.MemDBEntityByID(te.EntityID, false)
+		if err != nil {
+			return nil, err
+		}
+
+		if entity == nil {
+			entity, err = c.identityStore.MemDBEntityByMergedEntityID(te.EntityID, false)
```

Comment:
I'm wondering if it makes sense to move this into `c.identityStore.MemDBEntityByID` since once they are merged it should be transparent to the consumer of the identity store.

Reply:
Even better, write a helper function that given a token entry gets the full list of policies and have this use that. :-)

Reply:
Added a helper function.